docs/content/contributing/q-and-a.md

```diff
@@ -79,7 +79,7 @@ You can propose a new module [here][ModuleProposal].
 
 ### How long will it take for someone to respond and a module to be created/updated and published?
 
-Whilst there are SLAs defined for providing [support][ModuleSupport] for existing modules, there are currently no SLAs in place for the creation of new modules. The AVM core team is a small team and is currently working on automating the module creation process to make it as easy as possible for module owners to create and publish modules on their own.
+While there are SLAs defined for providing [support][ModuleSupport] for existing modules, there are currently no SLAs in place for the creation of new modules. The AVM core team is a small team and is currently working on automating the module creation process to make it as easy as possible for module owners to create and publish modules on their own.
```

Comment:
Why this change? I think `whilst` is intentional. Same for other occurrences

Reply:
As it turns out it's UK English, whereas we write our documentation in US English. 😊

Reply:
Goodness, gracious, me